linters/tslint.json

```diff
@@ -0,0 +1,154 @@
+{
+  "extends": ["tslint-eslint-rules"],
+  "rules": {
+
+    // RECOMMENDATIONS
+
+    "interface-name": [true, "always-prefix"],    // Require interface names to begin with a capital ‘I’
```

Comment:
We may want to consider allowing non-prefixed interfaces by default - it would be fair to allow implementer discretion on this rule.